4788/src/main/cpp/ControlMap.cpp

```diff
@@ -0,0 +1,55 @@
+#include "ControlMap.h"
+
```

Comment:
We can add to controlmap.h and use `constexpr val` instead for all constant ports and controls rather than separating it into two files cpp/h